[2023-08-22 14:11:47,932] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: new_etl.extraer_data manual__2023-08-22T14:11:46.018749+00:00 [queued]>
[2023-08-22 14:11:47,958] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: new_etl.extraer_data manual__2023-08-22T14:11:46.018749+00:00 [queued]>
[2023-08-22 14:11:47,959] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-08-22 14:11:47,959] {taskinstance.py:1377} INFO - Starting attempt 1 of 4
[2023-08-22 14:11:47,960] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-08-22 14:11:47,980] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extraer_data> on 2023-08-22 14:11:46.018749+00:00
[2023-08-22 14:11:47,993] {standard_task_runner.py:52} INFO - Started process 4618 to run task
[2023-08-22 14:11:48,002] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'new_etl', 'extraer_data', 'manual__2023-08-22T14:11:46.018749+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/etl2.py', '--cfg-path', '/tmp/tmp7rqf3aks', '--error-file', '/tmp/tmpicp_g6zy']
[2023-08-22 14:11:48,004] {standard_task_runner.py:80} INFO - Job 143: Subtask extraer_data
[2023-08-22 14:11:48,099] {task_command.py:371} INFO - Running <TaskInstance: new_etl.extraer_data manual__2023-08-22T14:11:46.018749+00:00 [running]> on host 0110d0e687e9
[2023-08-22 14:11:48,153] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Sol
AIRFLOW_CTX_DAG_ID=new_etl
AIRFLOW_CTX_TASK_ID=extraer_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-22T14:11:46.018749+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-22T14:11:46.018749+00:00
[2023-08-22 14:11:49,776] {python.py:173} INFO - Done. Returned value was: None
[2023-08-22 14:11:49,830] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=new_etl, task_id=extraer_data, execution_date=20230822T141146, start_date=20230822T141147, end_date=20230822T141149
[2023-08-22 14:11:49,897] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-22 14:11:50,016] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
